prompting/validators/tasks.py

```diff
@@ -100,9 +100,10 @@ def compose_prompt(self) -> str:
         criteria_bullet_points_str = "\n".join(criteria_bullet_points)
 
         prompt_template = textwrap.dedent(
-            """\
+        """\
         Read the preceding context delimited with triple backticks carefully.
-        Your task is to provide a step-by-step answer to the last question found in the text, elaborating on your thought process:
+        Your task is to provide a a clear and direct answer to the last question found in the text.        
```

Comment:
I think this is a great change! You may want to cut the extra 'a' between provide and clear though.

Reply:
Good catch! I will update it in a bit